libraries/Microsoft.Bot.Schema/Teams/BotMeetingNotificationBase.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Schema.Teams
+{
+    /// <summary>
+    /// Specifies Bot meeting notification base including channel data and type.
+    /// </summary>
+    public class BotMeetingNotificationBase
```

Comment:
I see here you decided to make BotMeetingNotificationBase as a concrete class with a protected constructor while you made BotMeetingNotification as an abstract class. 

Can we be consistent here in terms of the pattern?

Reply:
I see you are using the approach of using a concrete class with a protected constructor in other places. If there is no strong reason to use an abstract class in BotMeetingNotification, let's use a concrete class with a protected constructor approach across all model classes.

Reply:
Thanks for pointing it out. Updated.